### PR TITLE
Add SharedTree fuzz tests to stress pipeline

### PR DESCRIPTION
**Pull request by daesunp**
## Description

This PR adds the sharedTree fuzz tests to the stress pipeline

### CODE REVIEW

File: experimental/dds/tree2/src/test/shared-tree/fuzz/topLevelFuzz.spec.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-import { takeAsync } from "@fluid-internal/stochastic-test-utils";
+import { describeFuzz, takeAsync } from "@fluid-internal/stochastic-test-utils";
```
Comment (Abe27342) [inline experimental/dds/tree2/src/test/shared-tree/fuzz/topLevelFuzz.spec.ts:5]: none of the .ts file changes you have should be necessary. the fuzz harness already uses `describeFuzz` and you don't use the arguments from within the test suite for anything meaningful (like changing the test config).

What you *do* need that you don't have here is changes to the pipeline file: tools\pipelines\test-dds-stress.yml.

Comment (Abe27342) [reply experimental/dds/tree2/src/test/shared-tree/fuzz/topLevelFuzz.spec.ts:5]: if you want some more details, it's probably faster to explain in person--just let me know :)

Comment (daesunp) [reply experimental/dds/tree2/src/test/shared-tree/fuzz/topLevelFuzz.spec.ts]: reverted changes to `.ts` files and updated `tools\pipelines\test-dds-stress.yml`

File: experimental/dds/tree2/.eslintrc.js [hunk 2]
```diff
@@ -28,6 +28,7 @@ module.exports = {
 			files: ["src/test/**/*"],
 			rules: {
 				"@typescript-eslint/no-unused-vars": ["off"],
+				"import/no-internal-modules": "off",
```
Comment (Abe27342) [inline experimental/dds/tree2/.eslintrc.js:31]: should be unnecessary now

**Comment by Abe27342** [comment]
these changes look good. Re: "is there a way to test," the intended way to do that is using `test` branches in this repo (i.e. not your fork of it). You push these changes to a branch under the 'test' folder, run the client build pipeline for packages/artifacts to get published (they're published to a test feed rather than the usual location), then run the DDS stress flow.

I think there might be some barriers with how feeds are set up which prevent that flow from working right now, but we should fix them anyway as it should ideally be low-friction to test changes to these pipelines. If you want to go through the exercise with me it'd probably be good for infra health...

File: experimental/dds/tree2/.eslintrc.js (hunk 2 above)
Comment (daesunp) [reply experimental/dds/tree2/.eslintrc.js]: removed `"import/no-internal-modules": "off"`

**Review by Abe27342 — APPROVED** [review]

**Pull request merged**